docs/recipes/inspect_runtime.py

```diff
@@ -37,7 +37,8 @@
         # get topic types
         topic_to_type = ros.topic_to_type
         print(f"Topic to type: {topic_to_type}")
-
+        print("rosnodes:")
```

Comment:
We probably want to add a whole bunch of ROS2 recipes at some point, but that's another issue. We can retain the original recipe for now.